src/index.js

```diff
@@ -101,6 +98,13 @@ export default class OasisApp {
     return chunks;
   }
 
+  async signGetChunks(path, context, message) {
+
+    const serializedPath = await this.serializePath(path);
+    return prepareChunks(serializedPath, Buffer.from(context), Buffer.from([context.length]), Buffer.from(message))
```

Comment:
Now that we are reworking this a bit, it would be good to check that
- 0 <= context.length < 256
- and that later when calling concat `contextSizeBuffer` length is 1 byte